test/Passes/tpp-identity.mlir

```diff
@@ -0,0 +1,21 @@
+//RUN: tpp-opt %s --default-tpp-passes | tpp-run -e main  -entry-point-result=void | FileCheck %s
+func.func @myfunc(%in: tensor<4x4xf32>, %out: tensor<2x2x2x2xf32>) ->  tensor<2x2x2x2xf32> {
+  %1 = tensor.pack %in inner_dims_pos = [0, 1] inner_tiles = [2,2] into %out : tensor<4x4xf32> -> tensor<2x2x2x2xf32>
+  return %1 : tensor<2x2x2x2xf32>
+}
+
+
+func.func @main(){
+  %0 = arith.constant dense<[[0.0, 1.0, 2.0, 3.0],
+                               [4.0, 5.0, 6.0, 7.0],
+                               [8.0, 9.0, 10.0, 11.0],
+                               [12.0, 13.0, 14.0, 15.0]]> : tensor<4x4xf32>
+  %1 = tensor.empty():tensor<2x2x2x2xf32>
+  %c0 = arith.constant 0 : index
+  %2 = call @myfunc(%0,%1) : (tensor<4x4xf32>, tensor<2x2x2x2xf32>) -> tensor<2x2x2x2xf32>
+  %d1 = arith.constant -1.0 : f32
+  %v0 = vector.transfer_read %1[%c0, %c0, %c0, %c0], %d1 : tensor<2x2x2x2xf32>, vector<2x2x2x2xf32>
+  // CHECK: ( ( ( ( 0, 1 ), ( 4, 5 ) ), ( ( 2, 3 ), ( 6, 7 ) ) ), ( ( ( 8, 9 ), ( 12, 13 ) ), ( ( 10, 11 ), ( 14, 15 ) ) ) )
```

Comment:
You can avoid the print by comparing to another constant.

Would also be good to get some more variations on the tests.